library/coretests/tests/floats/f32.rs

```diff
@@ -203,92 +181,6 @@ fn test_classify() {
     assert_eq!(1e-38f32.classify(), Fp::Subnormal);
 }
 
-#[test]
-fn test_floor() {
-    assert_biteq!(f32::math::floor(1.0f32), 1.0f32);
```

Comment:
The f32 and f64 tests used explicit functions in the `math` module, but the f16 an f128 tests just called the methods. @tgross35 any idea what is up with that?

Reply:
Seems like this may have to do with moving these functions to core? I think for the integration tests here, it doesn't matter whether we call them through core or std.

Reply:
That's correct, I added them that way to make sure we were testing the right version but there shouldn't be any problem using the `std` version now.